docs/src/pages/components/menus/BasicMenu.js

```diff
@@ -17,7 +17,7 @@ export default function BasicMenu() {
     <div>
       <Button
         id="basic-button"
-        aria-controls="basic-menu"
+        aria-controls={open ? 'basic-menu' : undefined}
```

Comment:
👍 Looks correct based on https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-controls

> A combobox element has aria-controls set to a value that refers to the element that serves as the popup. The aria-controls **only needs to be set when the popup is visible**, but it is valid and easier to program to reference an element that is not visible.

Reply:
I guess that in our case, it's more than *non-visible* => *non-existent*.

On a different note, the thread in https://github.com/w3c/aria/issues/995#issuecomment-507382249 suggests that the attribute is close to [poop](https://heydonworks.com/article/aria-controls-is-poop/).